buildSrc/src/main/kotlin/Woraround.kt

```diff
@@ -0,0 +1,3 @@
+import java.util.Properties
+
+fun createProperties() : Properties = Properties()
```

Comment:
Why not writing `Properties()` or `java.util.Properties()` instead of `createProperties()`?

Reply:
It's a workaround. java.util.Properties not visible in build.gradle.kts

Reply:
Strange, I checked yesterday and it worked. Check out [this file](https://github.com/gradle/gradle/blob/master/buildSrc/build.gradle.kts), for example.